osu.Game.Rulesets.Catch/UI/CatcherArea.cs

```diff
@@ -411,6 +401,24 @@ public void Explode()
                     f.Expire();
                 }
             }
+
+            private class CatcherSprite : Sprite
+            {
+                public CatcherSprite()
+                {
+                    Size = new Vector2(CATCHER_SIZE);
+                    FillMode = FillMode.Fill;
```

Comment:
I mean the old code had the same issue but you should look into it because this is probably not intended.

Reply:
Last I checked it was working as expected.

Reply:
So I guess this is not being looked at further. I'm going to go ahead and merge this because the previous implementation had the same issue as far as I can see.